src/inspector/models/text/textsettingsmodel.cpp

```diff
@@ -597,6 +651,73 @@ void TextSettingsModel::updateIsLineSpacingAvailable()
     setIsLineSpacingAvailable(available);
 }
 
+void TextSettingsModel::updateIsPositionAvailable()
+{
+    bool available = true;
+    for (EngravingItem* item : m_elementList) {
+        if (item->parent()->isTextLineBaseSegment()) {
+            available = false;
+            break;
+        }
+    }
+
+    setIsPositionAvailableChanged(available);
+}
+
+void TextSettingsModel::propertyChangedCallback(const engraving::Pid propertyId, const QVariant& newValue)
+{
+    setPropertyValue(m_elementList, propertyId, newValue);
+
+    switch (propertyId) {
+    case Pid::BEGIN_FONT_FACE:
+        setPropertyValue(m_elementList, Pid::CONTINUE_FONT_FACE, newValue);
+        setPropertyValue(m_elementList, Pid::END_FONT_FACE, newValue);
```

Comment:
I wonder if it would be more solid to handle this in the setProperty method. That would guarantee that any time anybody sets Pid::BEGING_FONT_... you also set CONTINUE_FONT_... and END_FONT_....

Reply:
Ah, maybe that's not possible because of back compatibility when reading older files.